src/plugins/core/conditional_format.ts

```diff
@@ -60,6 +61,35 @@ export class ConditionalFormatPlugin
 
   loopThroughRangesOfSheet(sheetId: UID, applyChange: ApplyRangeChange) {
     for (const rule of this.cfRules[sheetId]) {
+      if (rule.rule.type === "DataBarRule" && rule.rule.rangeValues) {
+        const change = applyChange(rule.rule.rangeValues);
+        switch (change.changeType) {
+          case "REMOVE":
+            this.history.update(
+              "cfRules",
+              sheetId,
+              this.cfRules[sheetId].indexOf(rule),
+              "rule",
+              //@ts-expect-error
```

Comment:
`rangeValues` does not exist on the union of the `ConditionalFormatRuleInternal`. Do not know how to fix it